base/dict.jl

```diff
@@ -68,12 +68,65 @@ Dict{String,Int64} with 2 entries:
 
 Alternatively, a sequence of pair arguments may be passed.
 
-```jldoctest
-julia> Dict("A"=>1, "B"=>2)
+```jldoctest dictexamples
+julia> d = Dict("A"=>1, "B"=>2)
 Dict{String,Int64} with 2 entries:
   "B" => 2
   "A" => 1
 ```
+
+After creation of a Dict you can use `d[x]` for getting the values and `d[x]=y` for changing the values.
```

Comment:
This could perhaps use https://github.com/JuliaLang/julia/pull/34226 because of the docstring is getting very long now.